evergreen.yaml

```diff
@@ -23,18 +24,33 @@ variables:
 #              Functions              #
 #######################################
 functions:
+  git-config-windows-api-length:
+    command: shell.exec
+    type: setup
+    params:
+      silent: true
+      script: |
+        if [ "Windows_NT" == "$OS" ]; then
+          git config --system core.longpaths true
+        fi
```

Comment:
can we do this in the makefile and leave out the --system

Reply:
this is temporary anyway, I am going to put a TODO now with the build ticket number